internal/templatefn/fn.go

```diff
@@ -134,14 +135,32 @@ func All(siriusPublicURL, prefix, staticHash string) map[string]interface{} {
 }
 
 type CaseTabData struct {
-	Lpa     sirius.DigitalLpa
-	TabName string
+	Lpa               sirius.DigitalLpa
+	SortedLinkedCases []linkedCase
+	TabName           string
+}
+
+type linkedCase struct {
+	UID     string
+	Subtype string
 }
 
 func caseTab(lpa sirius.DigitalLpa, tabName string) CaseTabData {
+	var linkedCases []linkedCase
+	linkedCases = append(linkedCases, linkedCase{lpa.UID, lpa.Subtype})
+
+	for _, linkedLpa := range lpa.LinkedCases {
+		linkedCases = append(linkedCases, linkedCase{linkedLpa.UID, linkedLpa.Subtype})
+	}
+
+	sort.Slice(linkedCases, func(i, j int) bool {
+		return linkedCases[i].UID < linkedCases[j].UID
+	})
```

Comment:
Makes sense, though we might want to order by something created date to avoid the order changing (as new UIDs are randomly generated, a new case could be inserted between two old ones)